### ISSUE TITLE
calling super on symbol method blowsup

### ISSUE THREAD

**Issue by jsg2021** [babel/babel#984]
```
const sym = Symbol();

class Foo {
  [sym] () {
    return 1;
  }
}

class Bar extends Foo {
  [sym] () {
    return super[sym]() + 2;
  }
}

let i = new Bar();

console.log(i[sym]()); //<--boom
```

**Comment by neVERberleRfellerER** [comment]
With iojs (1.5.0 tested,but probably all versions) and node 0.12 when `Symbol` (is is native in these versions) is used in `Object.defineProperty`, it is non-enumerable and when it is explicitly set as enumerable, it is not respected and result is still non-enumerable. Since babel uses for-in loop to add properties to class prototype from temporal object, it fails.

**Comment by sebmck** [comment]
Added bonus of computed classes now having very nice output.

**Comment by sebmck** [comment]
Fixed as of 4.7.5, thanks!

**Comment by neVERberleRfellerER** [comment]
@sebmck Beautiful solution. Thank you.

**Comment by jsg2021** [comment]
awesome :)

**Comment by jsg2021** [comment]
er... with non-native Symbols its blowing up. Run the example above in the https://babeljs.io/repl on Safari or another browser w/o Symbols.

**Comment by jsg2021** [comment]
@sebmck I don't see a way to re-open this issue on my side.

**Comment by sebmck** [comment]
Fixed as of 4.7.8 :dancer:
